packages/react-core/src/components/CodeBlock/__tests__/CodeBlockAction.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { CodeBlockAction } from '../CodeBlockAction';
+
+test('CodeBlockAction renders', () => {
+  render(<CodeBlockAction data-testid="code-block-action">action</CodeBlockAction>);
```

Comment:
We can remove the testis attributes from these tests since they're not being used

```suggestion
  render(<CodeBlockAction>action</CodeBlockAction>);
```